static/app/views/settings/organizationAuditLog/auditLogList.tsx

```diff
@@ -51,6 +54,105 @@ const addUsernameDisplay = (logEntryUser: User | undefined) => {
   return null;
 };
 
+function AuditNote({entry, orgSlug}: {entry: AuditLog; orgSlug: Organization['slug']}) {
+  const {projects} = useProjects();
```

Comment:
According to this comment, currently all projects are loaded

https://github.com/getsentry/sentry/blob/4319707cbd0d7783596fd4a4b7e707d55f97f629/static/app/utils/useProjects.tsx#L148